<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>company_profit</class>
 <widget class="QDialog" name="company_profit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_title_Report">
     <property name="minimumSize">
      <size>
       <width>301</width>
       <height>78</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>301</width>
       <height>78</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="ImgReport">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Recource.qrc">:/rec/img/report.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbReport">
        <property name="minimumSize">
         <size>
          <width>184</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>203</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string> Отчет Прибыли</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>335</width>
       <height>326</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>335</width>
       <height>326</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="leCount_orders"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbID_2">
        <property name="text">
         <string>Номер Компании</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbINN_2">
        <property name="text">
         <string>Количесво Заказов</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbName_2">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leName"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leSum"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Прибыль</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад </string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnReport_orders">
     <property name="text">
      <string>Подробнее</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTextEdit" name="teResult">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QTableWidget" name="twOrg"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recource.qrc"/>
 </resources>
 <connections/>
</ui>
